features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/advanced/AdvancedSettingsPresenter.kt

```diff
@@ -52,6 +63,61 @@ class AdvancedSettingsPresenter @Inject constructor(
         }
             .collectAsState(initial = Theme.System)
         var showChangeThemeDialog by remember { mutableStateOf(false) }
+
+        // List of PushProvider -> Distributor
+        var distributors by remember { mutableStateOf<List<Pair<PushProvider, Distributor>>>(emptyList()) }
+        var distributorNames by remember { mutableStateOf<List<String>>(emptyList()) }
```

Comment:
I'd use `derivedStateOf ` instead of changing both field in the `LaunchedEffect`

Reply:
Yeah, both calls are sync and initialized so I don't see why we need to make these values mutable states, just calculating them inside the remember for `distributors` one and `derivedStateOf` for `distributorNames` should be simpler.

Reply:
Good point, thanks. 97aa57baf7dc905dd17f259ccab3532531a0dd54 should be OK, just initialize data in `remember` (no need to use `derivedStateOf`). OK for you both?